documentation/user_docs/docs/manual/Multifit.rst

```diff
@@ -2,146 +2,185 @@
 Multifit
 ########
 
-Horace (and the parent utilities library, Herbert) comes with a rich and powerful fitting syntax that is common to the methods used to fit functions or models of S(Q,w) to one or more datasets. The documentation here is only meant to give an introduction and overview. For the full help, please use the Matlab documentation for the various fitting functions that can be obtained by using the ``doc`` command, for example ``doc d1d/multifit`` (for fitting function like Gaussians to d1d objects) or ``doc sqw/multifit_sqw`` (fitting models for S(Q,w) to sqw objects). It is strongly recommended that you use ``doc``, not ``help`` to explore how to use these methods, so that you can navigate between the numerous pages of documentation in the Matlab help window.
+Horace (and the parent utilities library, Herbert) comes with a rich and powerful fitting syntax that is common to the
+methods used to fit functions or models of S(Q, w) to one or more datasets. The documentation here is only meant to give
+an introduction and overview. For the full help, please use the Matlab documentation for the various fitting functions
+that can be obtained by using the ``doc`` command, for example ``doc d1d/multifit`` (for fitting function like Gaussians
```

Comment:
@oerc0122  and @abuts Not part of the review, but starting a discussion. If we aim to have a good and accessible set of docs, is a good idea to separate info into "in the docs" and "in the matlab help"? Happy to keep the info in the matlab help, but I think it should also be in the docs.